widget_driver/README.md

```diff
@@ -337,7 +337,46 @@ Easy...
     ```
 
 2. Then we just run the generator like we did before...
-3. After that we just need to hand the variable over to the `DriverProvider` and that's it. 🥳
+3. We should get a compiler warning in the generated file. To resolve that we just have to add the generated mixin `$<YourDriver>ProvidedPropertiesMixin` to our driver like this:
+
+    ```dart
+    @GenerateTestDriver()
+      class CoffeeDetailPageDriver extends WidgetDriver with $CoffeeDetailPageDriverProvidedPropertiesMixin {
+        ...
+      }
+    ```
+
+4. This requires us to override `updateProvidedProperties(...)` which gets called on state updates. That way we can respond to new values to our provided properties. (Technical Note: This is because the Driver does not get rebuilt on state updates. And a call to `notifyWidget()` is not necessary, this function gets called before the widget shows the new data.)
```

Comment:
I guess here we should also make sure to reference the new name of this method.
So not `updateProvidedProperties`, but instead the new name which was something like:
`updateDriverProvidedProperties`